utils.h

```diff
@@ -13,7 +13,9 @@
 
 #define IS_ALPHA_KEYCODE(code) ((code) >= KC_A && (code) <= KC_Z)
 
-uint16_t st_char_to_keycode(char c);
-char st_keycode_to_char(uint16_t keycode);
-void st_multi_tap(uint16_t keycode, int count);
-void st_send_key(uint16_t keycode);
+uint16_t    st_char_to_keycode(char c);
+char        st_keycode_to_char(uint16_t keycode);
+void        st_multi_tap(uint16_t keycode, int count);
+void        st_send_key(uint16_t keycode);
+int         st_max(int a, int b);
+int         st_min(int a, int b);
```

Comment:
Why can't we simply include QMK's `util.h` where ever we need `MIN` and `MAX`? I realize that slightly complicates testing outside of the QMK build process, but the testing environment can simply contain a copy of `util.h` that has the macros we need.

I think the consistency with other QMK code would be worth it. (Very very very slight firmware size reduction too).

Reply:
QMK's MIN/MAX macros are not safe against double evaluation, which can lead to hard to find bugs.
`define MIN(x, y) (((x) < (y)) ? (x) : (y))`  -> `MIN(function1(), function2())` would call one of these funcs twice.
I agree that we should reuse existing QMK things when possible, and provide fallbacks for the tester.

Reply:
Fair point.